app/view_models/AtsForms.scala

```diff
@@ -18,20 +18,14 @@ package view_models
 
 import play.api.data.Form
 import play.api.data.Forms._
-import utils.TAXSEnums.YearEnum
-import utils.validation.ConstraintUtil.CompulsoryEnumMappingParameter
-import utils.validation.ErrorMessagesUtilAPI._
-import utils.validation.MappingUtilAPI._
+
 
 object AtsForms {
 
-  private val year_compulsory = {
-    val question = CompulsoryEnumMappingParameter(
-      simpleFieldIsEmptyConstraintParameter("year", "ats.select_tax_year.required"),
-      YearEnum
-    )
-    compulsoryEnum(question)
-  }
+  val yearChoice = "year"
 
-  val atsYearFormMapping = Form(mapping("year" -> year_compulsory)(TaxYearEnd.apply)(TaxYearEnd.unapply))
+  //TODO look at the error handling logic to show error messsage
```

Comment:
Need a ticket for this?

Reply:
we needed to fix this as part of this ticket only
handled the error logic for form validation